src/script/auth/page/Root.tsx

```diff
@@ -61,6 +57,28 @@ import * as AuthSelector from '../module/selector/AuthSelector';
 import * as LanguageSelector from '../module/selector/LanguageSelector';
 import {ROUTE} from '../route';
 
+/**
+ *  Lazy loading components for routes, which can only be accessed by activated feature flags
```

Comment:
Are these components large and rarely used? lazy loading does improves initial load time however lazy loading does introduce some overhead. When a component is lazily loaded, an additional network request is made to fetch the component's code when it is needed. This introduces a delay as the code needs to be fetched, parsed, and executed before the component can be rendered.

in the conference one of the precentor showed how lazy loading actually worsened overall performance.

Reply:
These routes are loaded for everyone right now, but are only used in special circumstances.

I think it makes sense to have better performance for most of the users and only having a impact (if any) for the users that will see this route.

Reply:
This does not belong in this PR. Please extract those changes to another PR, this might have consequences on automation and we don't want to rush this kind of optimization